File: API/ContainerNinja.Contracts/DTO/ProductDetailsDTO.cs
﻿
namespace ContainerNinja.Contracts.DTO
{
    public class ProductDetailsDTO
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public long? WalmartId { get; set; }
        public string? WalmartLink { get; set; }
        public string? WalmartSize { get; set; }
        public string? WalmartItemResponse { get; set; }
        public string? WalmartSearchResponse { get; set; }
        public string? Error { get; set; }
        public float Size { get; set; }
        public float Price { get; set; }
        public bool Verified { get; set; }
        public int UnitType { get; set; }
        public int? ProductStockId { get; set; }
    }
}

File: Client/src/app/models/ProductDetailsDTO.ts
import { WalmartProductDTO } from "./WalmartProductDTO";

export class ProductDetailsDTO {
  id?: number;
  name?: string;
  walmartId?: number = undefined;
  walmartLink?: string = undefined;
  walmartSize?: string = undefined;
  walmartItemResponse?: string = undefined;
  walmartSearchResponse?: string = undefined;
  error?: string = undefined;
  size?: number = undefined;
  price?: number = undefined;
  verified?: boolean = undefined;
  unitType?: number = undefined;
  productStockId?: number = undefined;
}
